idseq_dag/steps/run_srst2.py

```diff
@@ -0,0 +1,34 @@
+import shutil
+
+from idseq_dag.engine.pipeline_step import PipelineStep
+import idseq_dag.util.command as command
+
+class PipelineStepRunSRST2(PipelineStep):
+    '''
+    srst2 is used to detect resistance genes.
+    See: https://github.com/katholt/srst2
+    '''
+    def run(self):
+        ''' Invoking srst2 '''
+        fwd_rd = self.input_files_local[0][0]
+        rev_rd = self.input_files_local[0][1]
+        # TODO: pass in database in additional_files field 
+        db_file_path = self.input_files_local[1][0]
+        db_name = "ARGannot_r2" # hardcode for now
+        # srst2 assumes database file is in current directory
+        shutil.copy2(db_file_path, db_name)
```

Comment:
the lines `db_name = "ARGannot_r2" ` and `shutil.copy2(db_file_path, db_name)` necessary? It seems like the location `db_file_path` is specified explicitly in the srst2 command below